circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/transcript/transcript.test.cpp

```diff
@@ -78,28 +88,30 @@ void perform_mock_verifier_transcript_operations(auto transcript)
 /**
  * @brief Test basic transcript functionality and check circuit
  * @details Implicitly ensures stdlib interface is identical to native
- * @todo(luke): Underlying circuit is nearly trivial until transcript implements hashing constraints
+ *
  */
-TEST(stdlib_honk_transcript, basic_transcript_operations)
+TEST(RecursiveHonkTranscript, InterfacesMatch)
 {
     Builder builder;
 
+    constexpr size_t LENGTH = 8; // arbitrary length of Univariate to be serialized
+
     // Instantiate a Prover Transcript and use it to generate some mock proof data
     ProverTranscript prover_transcript;
-    auto proof_data = generate_mock_proof_data(prover_transcript);
+    auto proof_data = generate_mock_proof_data<UltraFlavor, LENGTH>(prover_transcript);
 
     // Instantiate a (native) Verifier Transcript with the proof data and perform some mock transcript operations
     VerifierTranscript native_transcript(proof_data);
-    perform_mock_verifier_transcript_operations(native_transcript);
+    perform_mock_verifier_transcript_operations<UltraFlavor, LENGTH>(native_transcript);
 
     // Confirm that Prover and Verifier transcripts have generated the same manifest via the operations performed
     EXPECT_EQ(prover_transcript.get_manifest(), native_transcript.get_manifest());
 
     // Instantiate a stdlib Transcript and perform the same operations
     Transcript<Builder> transcript{ &builder, proof_data };
-    perform_mock_verifier_transcript_operations(transcript);
+    perform_mock_verifier_transcript_operations<UltraRecursiveFlavor, LENGTH>(transcript);
 
-    // Confirm that the native and stdlib transcripts have generated the same manifest
+    // Confirm that the native and stdlib verifier transcripts have generated the same manifest
     EXPECT_EQ(transcript.get_manifest(), native_transcript.get_manifest());
 
     // TODO(luke): This doesn't check much of anything until hashing is constrained in the stdlib transcript
```

Comment:
When I implemented the stdlib Transcript for Honk, I "stubbed out" the hashing portion, i.e. it does not currently lay down contraints to emulate hashing. I did this at Codys suggestion for expediency since it was a big pain to get the result to agree with the native hash (pedersen + blake3) and we are going to be changing the hash very soon anyway.

Reply:
Added the issue number and a clearer comment